PlutoHelperAgent/PreferencesWindowController.m

```diff
@@ -0,0 +1,214 @@
+//
+//  PreferencesWindowController.m
+//  PlutoHelperAgent
+//
+//  Created by Dave Allison on 26/02/2018.
+//  Copyright © 2018 Guardian News & Media. All rights reserved.
+//
+
+#import "PreferencesWindowController.h"
+
+@interface PreferencesWindowController ()
+
+@end
+
+@implementation PreferencesWindowController
+
+- (void)windowDidLoad {
+    [super windowDidLoad];
```

Comment:
this function seems to do a number of individual operations, with variable names like status5 etc.

Please could you split each of these operations into its own function, with a descriptive name; that will make it easier for any of us to debug it in the future.

Reply:
Done.